src/wallet/wallet.cpp

```diff
@@ -2800,7 +2800,7 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
                 nFeeNeeded = GetMinimumFee(*this, nBytes, coin_control, &feeCalc);
                 if (feeCalc.reason == FeeReason::FALLBACK && !m_allow_fallback_fee) {
                     // eventually allow a fallback fee
-                    strFailReason = _("Fee estimation failed. Fallbackfee is disabled. Wait a few blocks or enable -fallbackfee.").translated;
+                    strFailReason = _("Fee estimation failed. Fallbackfee is disabled. Wait a few blocks or enable -fallbackfee=<BTC/kB>.").translated;
```

Comment:
Tend to NACK

* The documentation (`-help` or `-?`) is already pretty clear on this being an amount and not a flag
* This is a translation string, so all translations are invalided, for little reason. Also, it complicates translations
* The underlying bug should be fixed. See #18225

Reply:
FWIW debug logs already do it:

```
src/qt/bitcoinstrings.cpp:137:QT_TRANSLATE_NOOP("bitcoin-core", "Invalid amount for -fallbackfee=<amount>: '%s'"),
src/wallet/wallet.cpp:3799:            error = strprintf(_("Invalid amount for -fallbackfee=<amount>: '%s'").translated, gArgs.GetArg("-fallbackfee", ""));
```